SU-SOE/soe_profile
# syncd profile styles with stanford profile

# READY FOR REVIEW

# Summary
- Stanford profile styles needed syncd with stanford_profile

# Review Needed By (Date)
- ASAP

# Urgency
- Critical

# Steps to Test

1. Check out this branch and verify that the styles from stanford_profile wysiwyg are avaialble(Drop cap etc)

# Affected Projects or Products
- Leleandd8 stack

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/D8CORE-1544
